### PR TITLE
Allow URL in search result preview to be live updated

### LINKED ISSUE #10756: Friendly URL not updated in SEO preview
**Describe the bug**
Title says it all

**To Reproduce**
1. Go to 'Catalog -> Products'
2. Click on 'SEO'
3. Scroll down to 'Friendly URL', change it
4. See that Friendy URL is not updated in the preview

**Screenshots**
If applicable, add screenshots or screenrecords to help explain your problem.

**Additionnal information**
PrestaShop version: 1.7.5.0
PHP version: N/A

### PR DESCRIPTION
**Pull request by Quetzacoalt91**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.5.x
| Description?  | Allow the URL in search preview to be live updated.
| Type?         | bug fix
| Category?     | BO
| BC breaks?    | Nope
| Deprecations? | Nope
| Fixed ticket? | Fixes #10756
| How to test?  | Update the field Friendly URL in a product form, SEO tab

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/10810)

### CODE REVIEW

File: admin-dev/themes/new-theme/js/product-page/index.js [hunk 1]
```diff
@@ -43,6 +43,18 @@ $(() => {
...
     serpComp.vm.$refs.serp.setUrl($('#product_form_preview_btn').data('redirect'));
+
+    const watchedMetaUrl = $('.serp-watched-url:input');
+    const initialValue = watchedMetaUrl.val();
+
+    // Because the url is in a data attribute never updated, we need a custom update
+    watchedMetaUrl.on('keyup change', (e) => {
+      serpComp.vm.$refs.serp.setUrl(
+        $('#product_form_preview_btn').data('redirect').replace(
+          initialValue, $('.serp-watched-url:input').val()
```
Comment (PierreRambaud) [inline admin-dev/themes/new-theme/js/product-page/index.js]: `$('.serp-watched-url:input')` is already loaded with `watchedMetaUrl`

**Review by PierreRambaud — DISMISSED** [review]

**Comment by marionf** [comment]
@Quetzacoalt91 
It doesn't work if I change language
https://drive.google.com/open?id=1k-ngsbEm5gd1AonGMrxX1OHpN6QUkl0j

**Comment by Quetzacoalt91** [comment]
@marionf I only have the URL for the default language, I can't implement it for the other languages yet. (The issue is the same for the name and description).

**Review by PierreRambaud — APPROVED** [review]

**Comment by PierreRambaud** [comment]
Thanks @Quetzacoalt91

**Pull request merged**